src/VolleyManagement.Backend/VolleyManagement.Services/Authentication/VolleyUserStore.cs

```diff
@@ -228,7 +228,7 @@ public Task AddLoginAsync(UserModel user, UserLoginInfo login)
         /// <returns>Task to await</returns>
         public Task RemoveLoginAsync(UserModel user, UserLoginInfo login)
         {
-            var loginToDelete = user.Logins.Find(l =>
+            var loginToDelete = (user.Logins.ToList()).Find(l =>
```

Comment:
Why to create another List instance?

Reply:
ICollection<T> does not support find, need explicit cast

Reply:
It's not a cast.  Find another way.

Reply:
ok